Content.Client/Administration/UI/Tabs/ObjectsTab/ObjectsTab.xaml.cs

```diff
@@ -161,9 +147,7 @@ private object GetComparableValue((string Name, NetEntity Entity) entity, Object
     private void HeaderClicked(ObjectsTabHeader.Header header)
     {
         if (_headerClicked == header)
-        {
             _ascending = !_ascending;
-        }
         else
```

Comment:
If the else has brackets, then leave the brackets on the if for consistency and readability.

Reply:
These should be put back.